website/util/time.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is unrelated code; possibly the result of the cherry-pick? Can these commits be easily removed?